### DISCUSSION TITLE
How to apply same class for two different breakpoint

### DISCUSSION THREAD

**Discussion by ks221197** [tailwindlabs/tailwindcss#9394] [Help]
Tailwind provides feature for [arbitrary values](https://tailwindcss.com/docs/margin#arbitrary-values),
 eg `<div class="m-[5px]" />` 
and with some resreach i found way to apply mutilpe css class for one brackppoint,
 eg  `<div class="lg: ( m-[5px] p-[6px] text-[78px] )" />`

Now the question is,
_how to apply same CSS class on multiple breakpoint_,
 like suppose i have three screen resolutions defined in tailwind configuration name **lg** , **md**and  **sm**, 
 and want to apply same CSS for both breakpoints.
 
Currently, I have to write like this,

 `<div class="lg: ( m-[5px] p-[6px] text-[78px] ) md: ( m-[5px] p-[6px] text-[78px] ) sm: ( m-[1px] p-0 text-[8px] )" />`
 
 Is there any way to write this without redundancy
 
 just silly:confused: eg like, 
  `<div class="lg: md: ( m-[5px] p-[6px] text-[78px] ) sm: ( m-[1px] p-0 text-[8px] )  md: ( text-[#000000] )" />`

**Comment by wongjn** [comment]
I'm not sure what documentation you read, but there is no such grouping feature like `lg: ( m-[5px] p-[6px] text-[78px] )` in Tailwind.

Also for the record, you don't need to specify multiple breakpoints for the same class: `md:m-[5px] lg:m[5px]` can be simplified down to `md:m-[5px]`. This is because [Tailwind follows a mobile-first approach](https://tailwindcss.com/docs/responsive-design#mobile-first) to it's media queries.
